SCloudRandomizer/MainViewController.m

```diff
@@ -227,7 +239,7 @@ - (void)playPauseSong
     }
 }
 
-// TODO: Change logic to get tracks based on SearchParams object
+// Get tracks based on SearchParams
 - (void)getTracks
```

Comment:
Another way we could do this is pass a url to getTracks and ask it to fetch the songs from that url. So that way getTracks does not need to know which url it needs to use, we can pass that responsibility to the caller (plus its more testable that way!)

Also we could
1) On first load, fetch the favorite tracks using this method instead of hardcoding the search term. (if there are no fav tracks, we could pop up another modal asking them to explicitly set a search term)
2) Construct url based on the search term and then use this method whenever user changes search term